multik-api/src/main/kotlin/org/jetbrains/kotlinx/multik/ndarray/operations/Add.kt

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2020-2021 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package org.jetbrains.kotlinx.multik.ndarray.operations
+
+import org.jetbrains.kotlinx.multik.api.mk
+import org.jetbrains.kotlinx.multik.api.ndarray
+import org.jetbrains.kotlinx.multik.ndarray.data.*
+
+public fun <T : Number> D1Array<T>.append(n: D1Array<T>): D1Array<T> {
```

Comment:
I missed this moment.
The method signature wants to be a supertype `MultiArray`, D1Array is alias for NDArray<T, D1>